com.hack_computer.OperatingSystemApp.json

```diff
@@ -0,0 +1,55 @@
+{
+  "app-id": "com.hack_computer.OperatingSystemApp",
+  "add-extensions": {
+      "com.hack_computer.Clippy.Extension": {
+          "version": "beta",
+          "directory": "clippy",
+          "no-autodownload": false,
+          "autodelete": true
+      }
+  },
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.36",
+  "sdk": "org.gnome.Sdk",
+  "command": "com.hack_computer.OperatingSystemApp",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--socket=pulseaudio",
+    "--device=dri",
+    "--own-name=com.hack_computer.OperatingSystemApp",
+    "--env=GTK3_MODULES=/app/clippy/lib/libclippy-module.so",
+    "--talk-name=com.hack_computer.Clubhouse",
+    "--talk-name=com.hack_computer.GameStateService",
+    "--talk-name=com.hack_computer.HackSoundServer",
+    "--talk-name=org.gnome.Shell",
+    "--talk-name=com.hack_computer.HackableAppsManager",
+    "--talk-name=com.hack_computer.hack"
+  ],
+  "modules": [
+    {
+      "name": "clippy-ext",
+      "buildsystem": "simple",
+      "build-commands": [
+        "mkdir /app/clippy"
+      ],
+      "sources": []
+    },
+    {
+      "name": "toyapp",
```

Comment:
is this the correct name?

Reply:
I've changed the name to the app. This manifest is generated from a template, so the name was the same for all "toy apps", but now the id is used so it's more descriptive.